CHANGELOG.md

```diff
@@ -6,9 +6,13 @@
 
 * Extracted part of the nREPL-agnostic functionality to `orchard`.
 * Added a profiling middleware.
+* Support for orchard var-query in apropos
+* Support for orchard var-query in test
```

Comment:
I guess here it should also be mentioned the new op `test-var-query`. We also list those in the README.